frontend/src/components/mail-verification/verification-success/styles.module.scss

```diff
@@ -0,0 +1,63 @@
+.bgImage {
+  position: relative;
+  z-index: 1;
+  display: flex;
+  align-items: center;
+  min-height: 100vh;
+  background-position: center center;
+  background-size: cover;
+}
+
+.bgImage::before {
+  content: "";
+  position: absolute;
+  top: 0;
+  left: 0;
+  z-index: -1;
+  width: 100%;
+  height: 100%;
+  background-color: var(--blue300);
+  opacity: 0.5;
+}
```

Comment:
These styles should be broght to some higher level and reused. Because I see that they are duplicafted